pkg/security/secl/compiler/eval/glob.go

```diff
@@ -20,6 +20,8 @@ type Glob struct {
 }
 
 func (g *Glob) contains(filename string) bool {
+	filename = strings.ReplaceAll(filename, "\\", "/")
```

Comment:
Isn't `\` a valid character on Linux?  So on linux this would break if the path had a `\` in it?

Reply:
yeah, this is why we are doing something a bit more clever in https://github.com/DataDog/datadog-agent/pull/24304